.github/workflows/performance-tests.yml

```
@@ -0,0 +1,385 @@
+          python -m pip install \
+            pytest \
+            pytest-cov \
+            pytest-xdist \
+            pytest-timeout \
+            pytest-html \
+            coverage \
+            black \
+            flake8 \
+            mypy \
+            isort
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)